pkg/securitycontext/util.go

```diff
@@ -124,6 +124,22 @@ func DetermineEffectiveSecurityContext(pod *v1.Pod, container *v1.Container) *v1
 	return effectiveSc
 }
 
+// DetermineEffectiveRunAsUser returns a UID from the provided pod's and
+// container's security context. Container's runAsUser take precedence in cases
+// where both are set.
+func DetermineEffectiveRunAsUser(pod *v1.Pod, container *v1.Container) *int64 {
+	var runAsUser *int64
+	if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.RunAsUser != nil {
+		runAsUser = new(int64)
+		*runAsUser = *pod.Spec.SecurityContext.RunAsUser
+	}
+	if container.SecurityContext != nil && container.SecurityContext.RunAsUser != nil {
```

Comment:
You need to check the container before the pod right?
> If set in both SecurityContext and PodSecurityContext, the value specified in SecurityContext takes precedence

Reply:
no. container will overwrite pod so it comes later